Libraries/src/Amazon.Lambda.AspNetCoreServer/APIGatewayProxyFunction.cs

```diff
@@ -37,6 +39,16 @@ public abstract class APIGatewayProxyFunction
         public bool EnableResponseLogging { get; set; }
 
         /// <summary>
+        /// If true then all response content will be Base64-encoded before being returned.
+        /// </summary>
+        public bool EnableBinaryResponseContent { get; set; }
+
+        /// <summary>
+        /// An optional set of Content-Types that will trigger encoding the response content in Base64.
+        /// </summary>
+        public IEnumerable<string> BinaryResponseContentTypes { get; set; }
```

Comment:
With this change, do you also want to do away with the `XXXContentType` property and only have the one method `AddXXXContentType()` as the sole interface point to this feature?